build/build.sh

```diff
@@ -11,12 +11,16 @@ INPUT_PATH=references/generated/all-sections.md
 
 # Make output directory
 mkdir -p output
+# Move images to output directory
+cp -R sections/images output/
```

Comment:
I think that would work probably. Would the user have to commit the images to the output branch and the main text to the master branch?